samples/host_verify/host.c

```diff
@@ -131,6 +132,55 @@ oe_result_t verify_report(
     return result;
 }
 
+oe_result_t verify_evidence(
+    const char* evidence_filename,
+    const char* endorsements_filename)
+{
+    oe_result_t result = OE_FAILURE;
+    uint8_t *evidence = NULL, *endorsements = NULL;
+    size_t evidence_size = 0, endorsements_size = 0;
+
+    if (read_binary_file(evidence_filename, &evidence, &evidence_size))
+    {
+        if (endorsements_filename &&
+            !read_binary_file(
+                endorsements_filename, &endorsements, &endorsements_size))
+            return OE_INVALID_PARAMETER;
+
+        const oe_policy_t* policies = NULL;
+        size_t policies_size = 0;
+        oe_claim_t* claims = NULL;
+        size_t claims_length = 0;
+
+        oe_verifier_initialize();
```

Comment:
Why is line 155 needed?  Shouldn't this be done automatically on first call to oe_verify_evidence()?
We should keep samples free of unnecessary code, as much as possible.

Reply:
(see below)